### ISSUE TITLE
persistenced requires root on GH200

### ISSUE THREAD

**Issue by sclarkson** [NVIDIA/nvidia-persistenced#11]
```
Aug 27 21:35:25 ubuntu nvidia-persistenced[1405]: Started (1405)
Aug 27 21:35:25 ubuntu nvidia-persistenced[1405]: NUMA: Enabling NUMA memory Auto-Online due to GPU requirement
Aug 27 21:35:25 ubuntu nvidia-persistenced[1405]: NUMA: Failed to open /sys/devices/system/memory/auto_online_blocks: Permission denied
Aug 27 21:35:25 ubuntu nvidia-persistenced[1405]: NUMA: Failed to enable NUMA memory Auto-Online
Aug 27 21:35:25 ubuntu nvidia-persistenced[1400]: nvidia-persistenced failed to initialize. Check syslog for more details.
Aug 27 21:35:25 ubuntu nvidia-persistenced[1405]: PID file unlocked.
Aug 27 21:35:25 ubuntu systemd[1]: nvidia-persistenced.service: Control process exited, code=exited, status=1/FAILURE
Aug 27 21:35:25 ubuntu nvidia-persistenced[1405]: PID file closed.
```

Current [documentation](https://github.com/NVIDIA/nvidia-persistenced/blob/39e71b2cb7542d14f5106043fb4c7a789428144b/nvidia-persistenced.1.m4#L53-L56) says it does not need root, as well as the [install script]( https://github.com/NVIDIA/nvidia-persistenced/blob/39e71b2cb7542d14f5106043fb4c7a789428144b/init/install.sh#L636) defaulting to a non-root user.

Could we get some clarity on whether it's intended to be mandatory to run as root, here? Especially since `auto_online_blocks` is already properly set in my case, and persistenced is trying to write it unconditionally.

**Comment by hv15** [comment]
We've run into this issue as well for our installation.

**Comment by sclarkson** [comment]
Fixed in [550.120](https://github.com/NVIDIA/nvidia-persistenced/commit/672680bebf25101d88b668429a9840fe2b8ab7b9) and [565.57.01](https://github.com/NVIDIA/nvidia-persistenced/commit/6cfc9d3f14e1d504aae44ef40fe0dba1a8f6de64).
